blockchain/blockchain.go

```diff
@@ -50,6 +52,38 @@ type Blockchain struct {
 	Staged StagedBlock
 }
 
+func (b *Blockchain) RBCReceive([]byte) error{
```

Comment:
why Blockchain leader/follower is needed?

Reply:
this PR was created 10 days when we discussed meta struct. it is not meaningful now